README.md

```diff
@@ -153,6 +153,8 @@ information on its usage.
 | globalize/relative-time.js | 0.8KB | 0.6KB | [Relative time module](#relative-time-module) provides relative time formatting support |
 | globalize/unit.js | 0.9KB | 0.5KB | [Unit module](#unit-module) provides unit formatting support |
 
+Use [this page](http://johnnyreilly.github.io/globalize-so-what-cha-want/) for more details on which Cldrjs file & CLDR data to include.
+
```

Comment:
Can you move this sentence down too? Please, also consider using words parallel with https://github.com/jquery/globalize/pull/547/files#diff-04c6e90faac2675aa89e2176d2eec7d8R235.